polkadot/runtime/parachains/src/coretime/mod.rs

```diff
@@ -110,6 +110,11 @@ pub mod pallet {
 		/// Something that provides the weight of this pallet.
 		type WeightInfo: WeightInfo;
 		type SendXcm: SendXcm;
+
+		/// Maximum weight for any XCM transact call that should be executed on the coretime chain.
+		///
+		/// Basically should be `max_weight(set_leases, reserve, notify_core_count)`.
+		type MaxXCMTransactWeight: Get<Weight>;
```

Comment:
just a very very nit - to be aligned with the rest e.g. `MaxXcmEncodedSize`, `EnsureXcmOrigin`, `ExecuteXcm`, `SendXcm`...
```suggestion
		type MaxXcmTransactWeight: Get<Weight>;
```